doc/src/manual/noteworthy-differences.md

```diff
@@ -239,12 +239,12 @@ For users coming to Julia from R, these are some noteworthy differences:
   * Julia uses `nothing` of type `Nothing` to represent a null value, whereas Python uses `None` of type `NoneType`.
   * In Julia, the standard operators over a matrix type are matrix operations, whereas, in Python, the standard operators are element-wise operations. When both `A` and `B` are matrices, `A * B` in Julia performs matrix multiplication, not element-wise multiplication as in Python. `A * B` in Julia is equivalent with `A @ B` in Python, whereas `A * B` in Python is equivalent with `A .* B` in Julia.
   * The adjoint operator `'` in Julia returns an adjoint of a vector (a lazy representation of row vector), whereas the transpose operator `.T` over a vector in Python returns the original vector (non-op).
-  * In Julia, a function may contain multiple concrete implementations (called *Methods*), selected via multiple dispatch, whereas functions in Python have a single implementation (no polymorphism).
+  * In Julia, a function may contain multiple concrete implementations (called *Methods*), selected via multiple dispatch, whereas standard functions in Python have a single implementation (no polymorphism), although Python's Standard Library does include single-dispatch capability.
```

Comment:
```suggestion
  * In Julia, a function may contain multiple concrete implementations (called *methods*), which are selected via multiple dispatch based on the types of all arguments to the call, as compared to functions in Python, which have a single implementation and no polymorphism (as opposed to Python method calls which use a different syntax and allows dispatch on the receiver of the method).
```
I don't think it is really necessary to mention some extension implemented in the standard library: this is not a Python reference document and fundamentally Python _as a language_ does not support dispatch in function calls. The fact that you can simulate dispatch in function calls using other features of the language is interesting but not a necessary detail to bring up here. The fact that Python offer single dispatch natively via method calls seems much more relevant.